### PR TITLE
Task/propage entity name exp and timestamp type information

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

**Comment by fgalan** [comment]
Is there any relationship with PR https://github.com/telefonicaid/iotagent-node-lib/pull/1508 ?

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,3 @@
-- Fix: ensure service and subservice from device in logs about error proccesing message
+- Fix: ensure service and subservice from device in logs about error proccesing message
+- Fix: use entityNameExp with autoprovisioned devices (iota-node-lib#1145)
+- Fix: use timestamp from group autoprovisioned devices
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:3]: No associated issue?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:3]: No issue

**Comment by AlvaroVega** [comment]
> Is there any relationship with PR [telefonicaid/iotagent-node-lib#1508](https://github.com/telefonicaid/iotagent-node-lib/pull/1508) ?

Yes, it's related

**Review by mapedraza — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1 +1,3 @@
-- Fix: ensure service and subservice from device in logs about error proccesing message
+- Fix: ensure service and subservice from device in logs about error proccesing message
+- Fix: use entityNameExp with autoprovisioned devices (iota-node-lib#1145)
+- Fix: use but no store timestamp and entityNameExp from group with autoprovisioned devices
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:3]:
```suggestion
- Fix: use but not store timestamp and entityNameExp from group with autoprovisioned devices (iotagent-node-lib#1504, partially)
```

**Comment by fgalan** [comment]
> > Should a "twin" PR in IOTA-UL be done?
> 
> No twin PR for other agent is expected, it is just an expeciallitation of iotagent-node-library

However, maybe we have an issue here. Deferred to https://github.com/telefonicaid/iotagent-ul/issues/642

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**